sprint3/index.html

```diff
@@ -1,17 +1,26 @@
 <!DOCTYPE html>
 <html lang="ko">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta property="og:image" content="images/img_logo.png">
+  <meta property="og:title" content="판다 마켓">
+  <meta property="og:description" content="일상의 모든 물건을 거래해보세요">
+  <meta property="og:url" content="https://www.google.co.kr/?hl=ko">
   <title>판다마켓</title>
   <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/layout-home.css">
 </head>
+
 <body>
   <header class="header">
     <div class="header__logo">
       <a href="/" aria-label="홈으로 이동">
-        <img class="header__logo-img" src="images/img_logo.png" alt="판다마켓">
-        <img class="header__logo-img--mobile" src="images/img_logo_m.png" alt="판다마켓">
+        <picture>
+          <source media="(min-width: 767px)" srcset="images/img_logo.png">
+          <img class="header__logo-img" src="images/img_logo_m.png" alt="판다마켓" width="153" height="51">
+        </picture>
```

Comment:
picture태그를 사용하지않고도 화면 크기에 대응하는 이미지 옵션을 제공하고 현재 뷰포트 크기에 가장 적합한 이미지를 선택하게끔 만들 수 있습니다. 아래 코드처럼 바꿔볼까요?

```suggestion
        <img class="header__logo-img"
             src="images/img_logo_m.png"
             srcset="images/img_logo_m.png 153w, images/img_logo.png 198w"
             sizes="(max-width: 767px) 153px, 198px"
             alt="판다마켓" />
```